### PR TITLE
function app manual deployment instructions

### PR DESCRIPTION
**Pull request by v-atulyadav**
Required items, please complete
   
   Change(s):
   - See guidance below

   Reason for Change(s):
   - See guidance below

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - See guidance below

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

# Guidance <- remove section before submitting
-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

   Version updated:
   - Yes
   - Detections/Analytic Rule templates are required to have the version updated

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes/No/Need Help

_Note: If updating a detection, you must update the version field._

> Before the submission has been made, please look at running the KQL and Yaml Validation Checks locally.
> https://github.com/Azure/Azure-Sentinel#run-kql-validation-locally

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

File: DataConnectors/AzureFunctionsManualDeployment.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Function app manual deployment instructions 
+
+1. Deploy a Function App
+
+	NOTE: You will need to [prepare VS code](https://docs.microsoft.com/azure/azure-functions/functions-create-first-function-python#prerequisites) for Azure function development.
+
+	Download the [Azure Function App](https://aka.ms/sentinel-SlackAuditAPI-functionapp) file. Extract archive to your local development computer.
```
Comment (anki-narravula) [inline DataConnectors/AzureFunctionsManualDeployment.md:7]: This should be some generic placeholder, dont hard-code SlackAudit file

File: DataConnectors/AzureFunctionsManualDeployment.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+	- Start VS Code. Choose File in the main menu and select Open Folder.
+	
+	- Select the top level folder from extracted files.
+
```
Comment (anki-narravula) [inline DataConnectors/AzureFunctionsManualDeployment.md:12]: please add screenshots for each step

File: DataConnectors/AzureFunctionsManualDeployment.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+		e. **Select a runtime**: Choose Python 3.8.
+
+		f. Select a location for new resources. For better performance and lower costs choose the same [region](https://azure.microsoft.com/regions/) where Microsoft Sentinel is located.
+
+2. Deployment will begin. A notification is displayed after your function app is created.
+
+3. Deploy the function in Function app: Once the function app is created click on deploy button under workspace section. Select the Subcription and the function app in which function needs to be deployed.
+
+4. Go to Azure Portal for the Function App configuration.
```
Comment (anki-narravula) [inline DataConnectors/AzureFunctionsManualDeployment.md:35]: Abruptly ended, whats next after this ?

File: DataConnectors/AzureFunctionsManualDeployment.md (hunk 1 above)
Comment (v-atulyadav) [reply DataConnectors/AzureFunctionsManualDeployment.md]: Removed hard coded value

**Review by v-dvedak — APPROVED** [review]

**Pull request merged**